libs/models/elements/layout/images/src/lib/models-elements-layout-images.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class ModelsElementsLayoutImagesModule {}
```

Comment:
rename this to just ImagesModelsModule if the name was not provided

Reply:
Nevermind. This is a completely different issue. The changes shouldn't be part of this branch